src/main/java/io/supertokens/storageLayer/StorageLayer.java

```diff
@@ -80,18 +81,24 @@ public static Storage getNewStorageInstance(Main main, JsonObject config) throws
         }
         result.constructor(main.getProcessId(), Main.makeConsolePrintSilent);
```

Comment:
what happens if you create a new tenant and that tenant's db is down.. the API will fail, but it should not affect the console logs